### ISSUE TITLE
Writer: Paste using the Edit menu generates message "Your browser has very limited access to the clipboard"

### ISSUE THREAD

**Issue by SergioArbarviro** [CollaboraOnline/online#7658]
### Describe the Bug

The "Paste" function when using the Edit menu (but not the Ctl+V shortcut) does not work, and generates the message "Your browser has very limited access to the clipboard - lease use the following shortcuts: Ctl + C Ctl + X Ctl +V".

### Steps to Reproduce
In Writer:
1. Select text to copy or cut
2. Use the Edit "Copy" or "Cut" menu item (or the corresponding Ctl + C or Ctl + X shortcut)
3. Place the cursor to the place where you want to paste the text
4. Use the Edit>Paste option

### Expected Behavior

I expect the text in the clipboard to be pasted when using the Edit>Paste menu item.

### Actual Behavior

The text is not pasted, and a window opens with the message "Your browser has very limited access to the clipboard - lease use the following shortcuts: Ctl + C Ctl + X Ctl +V"

When using indeed the Ctl + V shortcut, the "paste" function works.

### Desktop

(Please complete the following information)
 - **OS:** Ubuntu 22.04
 - **Browser:** Firefox
 - **Version:** 119.0.1

**Comment by Tex2002ans** [comment]
Hey @SergioArbarviro, thanks for the report.

- Could you add your **Help > About** info too?

**Comment by SergioArbarviro** [comment]
Hello @Tex2002ans , I would be glad to address your request, but do not understand it. Where should I add this Help > About info? I see no such thing in my profile, nor in the description of the bug. Please be more specific, and I will be happy to answer to the best of my knowledge.

**Comment by Tex2002ans** [comment]
Hey @SergioArbarviro.

In Collabora Writer, if you go to the:

- "Help" tab

there should be an "About" button.

After you press that, you'll see:

- The exact Collabora Version + git hash + other helpful info.

There is even a little "Copy" button on that popup. If you press that and paste it here, that will help QA/devs know which exact version has this bug. :)

**Comment by SergioArbarviro** [comment]
Thank you @Tex2002ans for your guidance!

Here is the information that you requested:
Collabora Office 23.05.4.2 git hash: 
[78039c5]
Served by: Debian GNU/Linux 12 (bookworm) 
[ab034205]

**Comment by Ornanovitch** [comment]
I can confirm on 24.04

**Comment by Tex2002ans** [comment]
Hey @Ornanovitch . Thanks for confirming.

Can you also paste your exact **Help > About** information?

**Comment by Ornanovitch** [comment]
Of course :)

> Nextcloud Office
> Version COOLWSD :
> 24.04.3.1 git hash: 
> [594b605](javascript:void(window.open('https://github.com/CollaboraOnline/online/commits/594b605'));)
> Version LOKit :
> built on 
> [a great technology base](javascript:void(window.open('http://col.la/lot'));)
> Collabora Office 24.04.3.1 git hash: 
> [31b6e2c](javascript:void(window.open('https://hub.libreoffice.org/git-core/31b6e2c'));)
> Servi par : Debian GNU/Linux 11 (bullseye) 
> [90c556b2](https://office.felinn.org/hosting/discovery)

**Comment by thebearon** [comment]
Due to security reasons browser normally don't allow access to clipboard willy-nilly, so in general the message is correct. However, if you use a Chrome-based browser, there is a clipboard API that can be utilized to allow sites to access the clipboard at will, support for this has been added in CODE/COOL 24.04.

Blog posts on that:
https://vmiklos.hu/blog/sc-clipboard-paste.html
https://vmiklos.hu/blog/cool-clipboard-copy.html

In summary: if you see the bug in recent 24.04.3 or later versions when using Chrome, and after allowing clipboard access, it's a bug, otherwise expected.
